src/js/messaging/containers/Thread.jsx

```diff
@@ -147,19 +154,16 @@ export class Thread extends React.Component {
     // Exclude the current folder from the list of folders
     // that are passed down to the MoveTo component.
     const moveToFolders = [];
-    this.props.folders.forEach((folder) => {
+    folders.forEach((folder) => {
```

Comment:
`folders` is a Map, so it would need to be transformed to an Array first. I would like a one liner, but only if it does it in one pass.